packages/editor-ui/src/views/CanvasAddButton.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<div :class="$style.container" :style="containerCssVars" ref="container">
+		<n8n-tooltip placement="top" :value="showTooltip" :disabled="isScrimActive" popper-class="trigger-placeholder-button-tooltip">
```

Comment:
you can pass css module classes dynamically
```suggestion
		<n8n-tooltip placement="top" :value="showTooltip" :disabled="isScrimActive" :popper-class="$style.triggeTooltip">
```